modules/apps/headless-apio/parser/parser-apio-api/bnd.bnd

```diff
@@ -0,0 +1,3 @@
+Bundle-Name: Liferay Parser Apio API
```

Comment:
Parser is going to be renamed

![i-guarantee-it-hhhh](https://user-images.githubusercontent.com/340653/46738686-222c6000-cc9f-11e8-9a33-8f0fc9fcef15.jpg)